.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-2.6.3
```

Comment:
Can you use Ruby 2.7.x? (same as Foreman)

Reply:
Sure

Reply:
Fixed this issue in #11 which was merged 👍🏼